connector/connect/client/jvm/src/test/scala/org/apache/spark/sql/ClientE2ETestSuite.scala

```diff
@@ -1325,6 +1325,18 @@ class ClientE2ETestSuite extends RemoteSparkSession with SQLHelper with PrivateM
       .dropDuplicatesWithinWatermark("newcol")
     testAndVerify(result2)
   }
+
+  test("ES-1538905: DataFrameReader defaults to spark.sql.sources.default") {
```

Comment:
Should it be `SPARK-53054: DataFrameReader defaults to spark.sql.sources.default`?